services/src/unit-test/java/uk/gov/moj/sdt/services/RequeueOldIndividualRequestsTest.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.moj.sdt.services;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.moj.sdt.services.mbeans.SdtManagementMBean;
+import uk.gov.moj.sdt.utils.AbstractSdtUnitTestBase;
+
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+public class RequeueOldIndividualRequestsTest extends AbstractSdtUnitTestBase {
+
+    private RequeueOldIndividualRequests requeueOldIndividualRequests;
+
+    @Mock
+    private SdtManagementMBean sdtManagementMBean;
+
+    @BeforeEach
+    @Override
+    public void setUp() {
+        requeueOldIndividualRequests = new RequeueOldIndividualRequests(sdtManagementMBean);
+    }
+
+    @Test
+    void shouldInvokeRequeueOldIndividualRequests() {
+        requeueOldIndividualRequests.requeueOldIndividualRequests();
+        verify(sdtManagementMBean).requeueOldIndividualRequests(15);
```

Comment:
The expected value that `requeueOldIndividualRequests()` is called with may need to change once it's been made a property - see comment for `RequeueOldIndividualRequests` class.

Reply:
as this is Unit test case, we can set the values using setter method and expect the same value to be called.. updated the code. please review this again